modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -4,274 +4,198 @@
-|Required. Specifies the {op-system-first} images to download.
-|443, 80
-
-|`cloud.redhat.com/openshift`
-|Required. Used for cluster tokens.
-|443, 80
-
-|`registry.access.redhat.com`
-|Required. Used to access the `odo` CLI tool that helps developers build on OpenShift and Kubernetes.
-|443, 80
-
-|`quayio-production-s3.s3.amazonaws.com`
-|Required. Used to install and manage clusters in an AWS environment.
-|443, 80
-
-|`cm-quay-production-s3.s3.amazonaws.com`
-|Required. Used to install and manage clusters in an AWS environment.
-|443, 80
-
+|Domain | Port | Function
 |`ec2.amazonaws.com`
-|Required. Used to install and manage clusters in an AWS environment.
-|443, 80
+|443
+|Used to install and manage clusters in an AWS environment.
 
 |`events.amazonaws.com`
-|Required. Used to install and manage clusters in an AWS environment.
-|443, 80
+|443
+|Used to install and manage clusters in an AWS environment.
 
 |`iam.amazonaws.com`
-|Required. Used to install and manage clusters in an AWS environment.
-|443, 80
+|443
+|Used to install and manage clusters in an AWS environment.
 
 |`route53.amazonaws.com`
-|Required. Used to install and manage clusters in an AWS environment.
-|443, 80
+|443
+|Used to install and manage clusters in an AWS environment.
 
 |`sts.amazonaws.com`
-|Required. Used to install and manage clusters in an AWS environment.
-|443, 80
+|443
+|Used to install and manage clusters in an AWS environment.
 
-|`ec2.<aws_region>.amazonaws.com`
-|Required. Region dependent. Must be added per cluster and per region.
-|443, 80
+|`tagging.us-east-1.amazonaws.com`
+|443
+|Used to install and manage clusters in an AWS environment. This endpoint is always us-east-1, regardless of the region the cluster is deployed in.
 
-|`CLUSTER-NAME-k5bxz-image-registry-<aws_region>-lsiflffxtmfyikx.s3.dualstack.us-east-1.amazonaws.com`
-|Required. Region dependent. Must be added per cluster and per region.
-|443, 80
+|`ec2.<aws_region>.amazonaws.com`
+|443
+|Used to install and manage clusters in an AWS environment.
 
 |`elasticloadbalancing.<aws_region>.amazonaws.com`
-|Required. Region dependent. Must be added per cluster and per region.
-|443, 80
+|443
+|Used to install and manage clusters in an AWS environment.
 |===
-+
-Region is created during installation. To find the region, run:
-+
-[source,terminal]
-----
- $ rosa describe cluster --cluster=<cluster_name>
-----
-+
-To retrieve the endpoint, run:
-+
-[source,terminal]
-----
-$ oc -n openshift-image-registry get pod -l docker-registry=default -o json | jq '.items[].spec.containers[].env[] | select(.name=="REGISTRY_STORAGE_S3_BUCKET")'
-----
 
-. Operators require route access to perform health checks. Specifically, the authentication and web console Operators connect to two routes to verify that the routes work. If you are the cluster administrator and do not want to allow *.apps.<cluster_name>.<base_domain>, then you must allow these routes:
+. Allowlist the following OpenShift URLs:
```

Comment:
```suggestion
. Allowlist the following {product-title} URLs:
```
Should it be OpenShift or {product-title} here?

Reply:
This should not be {product-title}, as it is referencing OpenShift not ROSA.

Reply:
Thanks for clarifying @michaelryanmcneill -- please ignore this comment.